Alarmi/Alarmi/Sources/ViewControllers/TodayViewController.swift

```diff
@@ -10,12 +10,108 @@ import UIKit
 
 class TodayViewController: UIViewController {
 
+    // MARK: View
+    private let stackView: UIStackView = {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.axis = .vertical
+        $0.distribution = .fill
+        $0.alignment = .leading
+        $0.spacing = 8
+        return $0
+    }(UIStackView())
+
+    private let descriptionLabel: UILabel = {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.text = "마지막으로 전화한지 5일이 지났어요."
+        $0.font = UIFont.preferredFont(forTextStyle: .title2)
+        return $0
+    }(UILabel())
+
+    private let subDescriptionLabel: UILabel = {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.text = "2일 후면 전화하기로 한 날이에요."
+        $0.font = UIFont.preferredFont(forTextStyle: .body)
+        $0.textColor = .secondaryLabel
+        return $0
+    }(UILabel())
+
+    private let delayButton: UIButton = {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+
+        let config = UIImage.SymbolConfiguration(font: UIFont.preferredFont(forTextStyle: .caption2))
+        let image = UIImage(systemName: "chevron.right", withConfiguration: config)
+        $0.configuration?.image = image
+        $0.configuration?.imagePlacement = .trailing
+        $0.configuration?.baseBackgroundColor = .systemBlue
+        $0.configuration?.titleAlignment = .leading
+        $0.configuration?.contentInsets = .zero
+
+        var container = AttributeContainer()
+        container.font = UIFont.preferredFont(forTextStyle: .footnote)
+        $0.configuration?.attributedTitle = AttributedString("이번 연락 미루기", attributes: container)
```

Comment:
p1;

"이번 연락 미루기" 끝에 공백 추가해주세요.

```suggestion
        $0.configuration?.attributedTitle = AttributedString("이번 연락 미루기 ", attributes: container)
```

Reply:
adjustsFontForContentSizeCategory 에 반영헀습니다 : )